ONBOARDING.md

```diff
@@ -0,0 +1,121 @@
+- `utils`: contient tout type de constantes, de logiques métiers, fonctions pouvant être réutilisées à plusieurs endroits de l’application.
+- `pages`: contient la première couche de component qui est directement appelé après le routeur de l’application
+- `features` : contient les components propres à chacune des pages de l’application. Le rangement est presque partout cohérent avec leur utilité et les pages associées.
+- `hooks`: permet de rajouter des hooks customisés pour l’application.
+
+Le fichier contenant le routing de l’application (association entre url du navigateur et component à loader) se situe dans `App.tsx`.
+
+Les bases du thème (css) se situe dans `theme.css`.
+
+Toutes les pages de feature sont basées sur le même modèle. Tout d’abord, il y a les imports, puis les classes css propres au component qu’on met au maximum dans `sx` puis les hooks du component, et enfin le render.
+
+# Les choses utiles
+
+## Installation
+
+### Back
+
+- Cloner le projet
+- Lancer une image de docker postgres
+- Suivre la [doc du back](https://github.com/tournesol-app/tournesol/tree/main/backend)
+- Installer les dépendances de test dans les dossier `tests`
+- Faire le Setup d’une clé API pour discuter avec l’API Youtube
+
+### Front
+
+- Installer nvm 
+- Installer la version 14 `nvm install 14` et puis ` npm install -global yarn`
+- Créer un envrionement virtuel `mkvirtualenv {nom}`
+- Suivre la section Installation du front
+
+## Tips
+
+- Pour le backend, quand une modification est faite, il faut (toujours) écrire un test ou préciser un test existant. La commande `pytest` permet de lancer tous les tests.
+Cependant la commande `pytest --html=report.html –self-contained-html` permet de créer un fichier de rapport propre avec le détail du log séparé pour chacun des tests.
+
+- Pour mettre à jour le schéma d’OpenApi et plus précisément le `.yaml`, il faut d’abord lancer le back, puis dans le dossier du front tapez la commande `yarn run update-schema`.
+
+- Pour mettre à jour les fichiers dépendant du .yaml d’OpenApi, il faut lancer la commande suivante le dossier `scripts` du front : ` ./generate-services-from-openapi.sh`.
+
+- Pour corriger le style du code de manière automatique, il est possible d’activer un plugin selon l’IDE. Sinon on la commande `yarn lint:fix` permet dans tous les cas de corriger le code de manière automatique.
+
+# Fonctionnement de MR/projet
+
+Le suivi du projet et des tâches en cours se fait sur ce [board](https://github.com/tournesol-app/tournesol/projects/9)
+
+Il est possible de créer des idées qui nous semblent pertinentes en éditant une issue et en la plaçant dans la colonne `Backlog and ideas`.
+
+La core-team décide par la suite des tâches à prioriser et dépose les tickets correspondants dans la colonne `Ready`. On peut s’assigner sur un ticket après avoir discuté avec [Adrien][Adrien], [Romain][Romain]. ou [Louis][Louis].
```

Comment:
Nice!

If we translate this part, I'd recommend to add an invitation for the contributors to ask questions is the issue is not that clear, and maybe to agree with the team before starting to work if the task requires to choose between several options (as all options could have different consequences, more or less impacting).